app/src/main/java/com/duycuong/weather/ui/screen/main/nextdays_weather/ItemWeatherViewModel.java

```diff
@@ -0,0 +1,42 @@
+package com.duycuong.weather.ui.screen.main.nextdays_weather;
+
+import android.databinding.BaseObservable;
+
+import com.duycuong.weather.R;
+import com.duycuong.weather.data.model.Datum;
+import com.duycuong.weather.utils.Constant;
+import com.duycuong.weather.utils.IconWeather;
+import com.duycuong.weather.utils.TimeUtils;
+
+/**
+ * Created by DuyCương on 09/02/2018.
+ */
+
+public class ItemWeatherViewModel extends BaseObservable {
+
+    private Datum mDailyWeather;
+
+    ItemWeatherViewModel(Datum dailyWeather) {
+        mDailyWeather = dailyWeather;
+    }
+
+    public String getDate() {
+        return TimeUtils.covertTime(mDailyWeather.getTime());
+    }
+
+    public String getSummary() {
+        return mDailyWeather.getSummary();
+    }
+
+    public double getTemperatureHight() {
+        return mDailyWeather.getTemperatureHigh();
+    }
+
+    public double getTemperatureLow() {
+        return mDailyWeather.getTemperatureLow();
+    }
+
+    public int getIcon() {
+       return IconWeather.getStringIcon(mDailyWeather.getIcon());

```

Comment:
ERROR: 'method def' child have incorrect indentation level 7, expected level should be 8.